spec.emu

```diff
@@ -698,18 +698,12 @@ contributors: Ujjwal Sharma, Younies Mahmoud
           1. If _numberingSystem_ does not match the Unicode Locale Identifier `type` nonterminal, throw a *RangeError* exception.
         1. Let _opt_ be the Record { [[localeMatcher]]: _matcher_, [[nu]]: _numberingSystem_ }.
         1. Let _r_ be ResolveLocale(%Intl.DurationFormat%.[[AvailableLocales]], _requestedLocales_, _opt_, %Intl.DurationFormat%.[[RelevantExtensionKeys]], %Intl.DurationFormat%.[[LocaleData]]).
-        1. Let _locale_ be r.[[locale]].
-        1. Set _durationFormat_.[[Locale]] to _locale_.
-        1. Set _durationFormat_.[[DataLocale]] to _r_.[[dataLocale]].
-        1. Let _dataLocale_ be _durationFormat_.[[DataLocale]].
-        1. Let _dataLocaleData_ be _durationFormat_.[[LocaleData]].[[&lt;_dataLocale_&gt;]].
-        1. Let _digitalFormat_ be _dataLocaleData_.[[DigitalFormat]].
-        1. Let _twoDigitHours_ be _digitalFormat_.[[TwoDigitHours]].
-        1. Let _hourMinuteSeparator_ be _digitalFormat_.[[HourMinuteSeparator]].
-        1. Set _durationFormat_.[[HourMinuteSeparator]] to _hourMinuteSeparator_.
-        1. Let _minuteSecondSeparator_ be _digitalFormat_.[[MinuteSecondSeparator]].
-        1. Set _durationFormat_.[[MinuteSecondSeparator]] to _minuteSecondSeparator_.
-        1. Set _durationFormat_.[[NumberingSystem]] to _r_.[[nu]].
+        1. Set _durationFormat_.[[Locale]] to _r_.[[Locale]].
+        1. Let _resolvedLocaleData_ be _r_.[[LocaleData]].
+        1. Let _digitalFormat_ be _resolvedLocaleData_.[[DigitalFormat]].
+        1. Set _durationFormat_.[[HourMinuteSeparator]] to _digitalFormat_.[[HourMinuteSeparator]].
+        1. Set _durationFormat_.[[MinuteSecondSeparator]] to _digitalFormat_.[[MinuteSecondSeparator]].
+        1. Set _durationFormat_.[[NumberingSystem]] to _resolvedLocaleData_.[[nu]].
```

Comment:
Should this not be
```suggestion
        1. Set _durationFormat_.[[NumberingSystem]] to _r_.[[nu]].
```
still?

Reply:
Hmm, I'm confused now. I thought this was a bug in the spec and we needed to get the `[[nu]]` from the resolved locale data because of this information being stored on there (https://github.com/tc39/proposal-intl-duration-format/pull/229/files#diff-7d681727fcf47dc0b9a7512a470fb0da63276c625891a5cc232d725bd12912fdL852) but perhaps including this information is unnecessary like this and instead we should get rid of `[[nu]]` inside the locale data structure and use the `ResolveLocale` key resolution instead?

Reply:
As far as I can tell, the change in https://github.com/tc39/ecma402/pull/849 means that _r_ no longer has a [[dataLocale]] field, which is replaced by [[LocaleData]], but there's no change to the [[<_key_>]] fields for _key_ in %Intl.DurationFormat%.[[RelevantExtensionKeys]], which includes "nu".

Reply:
Pushed another commit addressing this as well as removing the `[[nu]]` stuff on the locale data. @anba could you also check the more recent commit and confirm that the `[[nu]]` was indeed redundant? I know this part of the spec is always a bit handwavy but wanted to know if things still look good from an implementer's perspective.

Reply:
`r.[[nu]]` contains the resolved numbering system, i.e. the numbering system selected either through Unicode extension keys or the `"numberingSystem"` option. 

Whereas `r.[[LocaleData]].[[nu]]` contains the list of numbering systems which are supported for the requested locale. That means `r.[[nu]] ∈ r.[[LocaleData]].[[nu]]`.